recipes/ninja/all/conanfile.py

```diff
@@ -49,6 +48,5 @@ def package_info(self):
         self.conf_info.define("tools.cmake.cmaketoolchain:generator", "Ninja")
 
         # TODO: to remove in conan v2
-        if Version(conan_version).major < 2:
-            self.env_info.PATH.append(os.path.join(self.package_folder, "bin"))
-            self.env_info.CONAN_CMAKE_GENERATOR = "Ninja"
+        self.env_info.PATH.append(os.path.join(self.package_folder, "bin"))
+        self.env_info.CONAN_CMAKE_GENERATOR = "Ninja"
```

Comment:
Should this be added to build env for the new generators?

Reply:
I would say more, it should configure `tools.cmake.cmaketoolchain:generator`

https://docs.conan.io/en/latest/reference/config_files/global_conf.html

Reply:
The recipe already defines the conf value for the generator. Should I additionally add it to build_env path?